docs/modules/superset/pages/usage-guide/security.adoc

```diff
@@ -126,6 +126,71 @@ Further information for specifying an AuthenticationClass for an OIDC provider c
 Superset has a concept called `Roles` which allows you to grant user permissions based on roles.
 Have a look at the {superset-security}[Superset documentation on Security^{external-link-icon}^].
 
+=== [[OPA]] OPA role mapping
+
+Superset can sync roles from OpenPolicyAgent.
+Currently only mapping is enabled as a larger refactoring of the upstream Superset concerning their security management is announced.
+
+In order to map roles from OPA into Superset, we expect rego rules with a rule name `user_roles`.
+In the below example two users `admin` and `testuser` have roles defined as rego rule in Rego v1 to be compliant with OPA v1.0.0 release.
+
+IMPORTANT: Only role mapping is enabled!
```

Comment:
Can you please mention that the responsibility to create the roles is up to the Superset admin?
And that an error will be logged and the role is not added to the user in this case?

Reply:
I [reworded](https://github.com/stackabletech/superset-operator/pull/582/commits/d69ff8b2325a5b1984240bada64f89d006c18cf9) the entire section.